packages/sonarwhal/src/lib/cli/rules/common.ts

```diff
@@ -224,7 +225,7 @@ export class NewRule implements INewRule {
         };
         this.useCase[ruleData.useCase] = true;
         this.prefix = type === QuestionsType.core ? '../../' : 'sonarwhal/dist/src/lib/';
-        this.worksWithLocalFiles = type !== QuestionsType.core;
+        this.scope = (type !== QuestionsType.core) ? Scope.any : Scope.site;
```

Comment:
This check is weird, especially once #806 is implemented. This could be a new question that pops depending on the type of rule the user has decided (interop, request, third party service, etc.) or we just ask it directly.